pymc3/sampling.py

```diff
@@ -265,20 +265,26 @@ def _choose_backend(trace, chain, shortcuts=None, **kwds):
     except KeyError:
         raise ValueError('Argument `trace` is invalid.')
 
+def _make_parallel(arg, njobs):
+    if not shape(arg):
```

Comment:
Its pretty robust, actually: empty lists, empty tuples, zero, even empty NumPy arrays all evaluate to `False` for conditional statements.